packages/react-server/src/ReactFlightServer.js

```diff
@@ -4360,6 +4360,7 @@ export function abort(request: Request, reason: mixed): void {
     if (request.destination !== null) {
       flushCompletedChunks(request, request.destination);
     }
+    callOnAllReadyIfReady(request);
```

Comment:
It seems better to duplicate this right after each `clear()` so that it's associated with the mutation that might have unblocked it and if there's ever a path that doesn't do a mutation then we don't have to check it in that branch.